.gitignore

```diff
@@ -4,6 +4,8 @@
 /node_modules
 /.pnp
 .pnp.js
+package-lock.json
+package.json
 
```

Comment:
I believe we want to check both  `package-lock.json` and `package.json` to our repository. This way we all share the same package related json files.

Reply:
fixed.. will push up a refactored pr